tests/conftest.py

```diff
@@ -468,3 +483,42 @@ def create_bucket(name, account_id, region_name):
         f"arn:aws:s3:{region_name}:{account_id}:bucket/{name}",
         creation_date,
     )
+
+
+def create_instance(image_id, instance_type, tags, region_name):
+    """Creates and instance using the given `image_id`, for instance:
+    'ami-one', `instance_type`, for instance: 't2.micro', `tags`, for instance
+    [{'Key': 'tag_one', 'value': 'value'}] and `region_name`, for instance:
```

Comment:
`[{'Key': 'tag_one', 'value': 'value'}]` seems a bit weird. Specially the `'value': 'value'` part. What about `[{'Key': 'tag_one', 'value': 'value_one'}]`?

Reply:
👍